doc/devel/contributing.rst

```diff
@@ -285,30 +285,57 @@ Coding guidelines
 API changes
 -----------
 
-Changes to the public API must follow a standard deprecation procedure to
-prevent unexpected breaking of code that uses Matplotlib.
-
-- Deprecations must be announced via a new file in
-  a new file in :file:`doc/api/next_api_changes/deprecations/` with
-  naming convention ``99999-ABC.rst`` where ``99999`` is the pull request
-  number and ``ABC`` are the contributor's initials.
-- Deprecations are targeted at the next point-release (i.e. 3.x.0).
-- The deprecated API should, to the maximum extent possible, remain fully
-  functional during the deprecation period. In cases where this is not
-  possible, the deprecation must never make a given piece of code do something
-  different than it was before; at least an exception should be raised.
-- If possible, usage of an deprecated API should emit a
-  `.MatplotlibDeprecationWarning`. There are a number of helper tools for this:
-
-  - Use ``_api.warn_deprecated()`` for general deprecation warnings.
-  - Use the decorator ``@_api.deprecated`` to deprecate classes, functions,
-    methods, or properties.
-  - To warn on changes of the function signature, use the decorators
-    ``@_api.delete_parameter``, ``@_api.rename_parameter``, and
-    ``@_api.make_keyword_only``.
-
-- Deprecated API may be removed two point-releases after they were deprecated.
-
+API consistency and stability are of great value. Therefore, API changes
+(e.g. signature changes, behavior changes, removals) will only be conducted
+if the added benefit is worth the user effort for adapting.
+
+All API changes in Matplotlib have to be performed following the below
+deprecation process. This ensures that users are notified before the change
+will take effect and thus prevents unexpected breaking of code.
+
+Deprecation process
```

Comment:
Don't introduce only one subsection of a section.  If you want subsections, suggest "Rules" "Introducing..." and "Expiring..."  I think it is obvious that these are the "deprecation process"

Reply:
Alright, changed the subsection titles and got rid of the obvious (to generic) title "Deprecation process"